django_rq/queues.py

```diff
@@ -49,6 +49,7 @@ class DjangoRQ(Queue):
     def __init__(self, *args, **kwargs):
         autocommit = kwargs.pop('autocommit', None)
         self._autocommit = get_commit_mode() if autocommit is None else autocommit
+        self.exception_handlers = kwargs.pop('exception_handlers', None)
```

Comment:
You can simply write a test for `get_queue('default')` and ensure that `queue.exception_handlers` contains the exception handler you specified in settings.